units/console-getty.service.in

```diff
@@ -20,10 +20,9 @@ Before=getty.target
 ConditionPathExists=/dev/console
 
 [Service]
-# The '-o' option value tells agetty to replace 'login' arguments with an
-# option to preserve environment (-p), followed by '--' for safety, and then
-# the entered username.
-ExecStart=-/sbin/agetty -o '-p -- \\u' --noclear --keep-baud - 115200,38400,9600 $TERM
+# The '-o' option value tells agetty to replace 'login' arguments with an option to preserve environment (-p),
+# followed by '--' for safety, and then the entered username.
+ExecStart=-/sbin/agetty -o '-p -- \\u' - $TERM
```

Comment:
/dev/console might well be a real serial port (i.e. on a server or embedded device that boots with console=ttyS0), we should leave the baud stuff in place here. it's ok to drop from the container getty though